code/modules/economy/requisition/rc_aid.dm

```diff
@@ -270,20 +271,10 @@ ABSTRACT_TYPE(/datum/rc_entry/item/surgical)
 	typepath = /obj/item/ai_interface
 	feemod = PAY_EMBEZZLED
 
-/datum/rc_entry/item/t_ray
-	name = "T-ray scanner"
-	typepath = /obj/item/device/t_scanner
-	feemod = PAY_TRADESMAN*2
-
-/datum/rc_entry/item/soldering
-	name = "soldering iron"
-	typepath = /obj/item/electronics/soldering
-	feemod = PAY_TRADESMAN*2
-
 /datum/rc_entry/item/multitool
 	name = "multitool"
 	typepath = /obj/item/device/multitool
-	feemod = PAY_TRADESMAN*4
+	feemod = PAY_EXECUTIVE
```

Comment:
these are easily orderable, not sure it needs the bonus pay

Reply:
Was on the fence about it. I don't mind it being swapped back if there's broad agreement.

Reply:
You know what, you're probably right.